tests/phpunit/tests/test-switch-blog-translatable-objects.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+if ( is_multisite() ) :
+
+	/**
+	 * Test class for registered translatable objects retrieval in multisite.
+	 */
+	class Switch_Blog_Translatable_Objects_Test extends PLL_Multisites_TestCase {
+		private $translatable_objects = array();
+
+		/**
+		 * Sets translatable objects then lets parent managing setup.
+		 *
+		 * @param WP_Site $blog      Blog to set up.
+		 * @param array   $languages Languages to create.
+		 * @param array   $options   Polylang options to use.
+		 * @param string  $structure Permalink structure to use.
+		 * @return void
+		 */
+		protected function set_up_blog_with_pll( WP_Site $blog, array $languages, array $options, string $structure ) {
+			$cpt = uniqid( 'cpt_', true );
+			$tax = uniqid( 'tax_', true );
+			$this->translatable_objects[ (int) $blog->blog_id ] = array(
```

Comment:
Casting array keys as integers is useless: php does it by itself.
```php
$arr = [ '7' => 'foo' ];
foreach ( $arr as $k => $v ) {
	var_dump( $k === "$k" ); // bool(false)
	var_dump( $k === (int) $k ); // bool(true)
}
```
So we can simplify the code of class a bit.

Reply:
👍 
Done in f2331030.